docs/chapters/04_features.md

```diff
@@ -761,4 +761,82 @@ boost nuke -e <environment name>
 
 For a force delete without asking for confirmation, you can run `boost nuke -e <environment name> -f`.
 
-> [!ATTENTION]  Be EXTRA CAUTIOUS with this option, all your application data will be irreversibly DELETED without confirmation.
\ No newline at end of file
+> [!ATTENTION]  Be EXTRA CAUTIOUS with this option, all your application data will be irreversibly DELETED without confirmation.
+
+## Events API
+
+Booster now supports fetching events from your Booster application! Here are some examples of how it works:
+
```

Comment:
I didn't put the HTTP request because we have more examples in other places and it looks pretty much the same. Also changed a bit the structure, let me know what you think @alvaroloes 🙏